fairing/notebook_helper.py

```diff
@@ -19,9 +18,9 @@ def get_notebook_name():
         for nn in json.loads(response.text):
             if nn['kernel']['id'] == kernel_id:
                 full_path = nn['notebook']['path']
-                return os.path.basename(full_path)
+                return full_path
```

Comment:
Have you tried this with Docker and Knative?
I am pretty sure this will break. For docker and knative builder we need the basename of the file not the fullpath.
* Either call `os.path.basename` where needed in `DockerBuilder` and `KnativeBuilder`
* Or create 2 functions in `notebook_helper.py`, i.e. `get_notebook_name` and `get_notebook_path`

Reply:
will revert to master, as we discussed, won't work.

Reply:
reverted to master.